plugins/com.google.cloud.tools.eclipse.appengine.localserver/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,7 @@ Bundle-Localization: plugin
 Bundle-RequiredExecutionEnvironment: JavaSE-1.7
 Bundle-Activator: com.google.cloud.tools.eclipse.appengine.localserver.Activator
 Bundle-ActivationPolicy: lazy
-Require-Bundle: com.google.cloud.tools.appengine;bundle-version="0.1.0",
+Require-Bundle: com.google.cloud.tools.appengine;bundle-version="0.1.14",
```

Comment:
Mea culpa: I'm running against source.  [0.1.13 is in the wild](http://search.maven.org/#artifactdetails%7Ccom.google.cloud.tools%7Cappengine-plugins-core%7C0.1.13%7Cjar)